Ladybird/Qt/BrowserWindow.cpp

```diff
@@ -402,8 +402,9 @@ BrowserWindow::BrowserWindow(Vector<URL::URL> const& initial_urls, WebView::Cook
     });
     QObject::connect(quit_action, &QAction::triggered, this, &QMainWindow::close);
     QObject::connect(m_tabs_container, &QTabWidget::currentChanged, [this](int index) {
-        setWindowTitle(QString("%1 - Ladybird").arg(m_tabs_container->tabText(index)));
-        set_current_tab(verify_cast<Tab>(m_tabs_container->widget(index)));
+        auto* tab = verify_cast<Tab>(m_tabs_container->widget(index));
+        setWindowTitle(QString("%1 - Ladybird").arg(tab->title()));
```

Comment:
This causes a crash after closing the last tab (i.e. rather than exiting via ctrl+c). We get a signal with the index set to -1, and end up with a null tab here. So maybe throw this at the top:
```c++
if (index < 0) {
    set_current_tab(nullptr);
    return;
}
```

Reply:
Along a similar line, I went with. Although if the short-circuit above is preferred, I'm happy to change :) 
```cpp
auto* tab = verify_cast<Tab>(m_tabs_container->widget(index));
if (tab)
    setWindowTitle(QString("%1 - Ladybird").arg(tab->title()));

set_current_tab(tab);
```